src/placeos-build/drivers.cr

```diff
@@ -277,7 +277,9 @@ module PlaceOS::Build
       Dir
         .glob(repository_path / "drivers/**/*.cr")
         .select! { |file|
-          !file.ends_with?("_spec.cr") && File.read_lines(file).any?(&.includes?("< PlaceOS::Driver"))
+          !file.ends_with?("_spec.cr") && File.read_lines(file).any? { |line|
+            line.includes?("< PlaceOS::Driver") && !line.includes?("abstract ")
```

Comment:
Just to sure up that we're matching classes

Reply:
I was thinking `"abstract-"` as otherwise `"abstract--class"` (two spaces) might be missed
and then we don't have to match as much so should execute a little bit faster